nomad/local/grapl-local-infra.nomad

```diff
@@ -428,6 +428,57 @@ job "grapl-local-infra" {
     }
   }
 
+  group "dnsmasq" {
+    network {
+      mode = "bridge"
+      port "dns" {
+        static = 53
+        to     = 53
+      }
+    }
+
+
+    task "dnsmasq" {
+      driver = "docker"
+
+      config {
+        image      = "andyshinn/dnsmasq"
```

Comment:
https://hub.docker.com/r/andyshinn/dnsmasq/ says we should use a different one

Reply:
I've fixed this, thanks for pointing this out